documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -2013,7 +2013,15 @@ The following configuration properties are _required_ unless a default value is
 
 |[[db2-property-database-server-name]]<<db2-property-database-server-name, `+database.server.name+`>>
 |No default
-|Logical name that identifies and provides a namespace for the particular Db2 database server that hosts the database for which {prodname} is capturing changes. Only alphanumeric characters, hyphens, dots and underscores must be used in the database server logical name. The logical name should be unique across all other connectors, since it is used as a topic name prefix for all Kafka topics that receive records from this connector.
+|Logical name that identifies and provides a namespace for the particular Db2 database server that hosts the database for which {prodname} is capturing changes.
+Only alphanumeric characters, hyphens, dots and underscores must be used in the database server logical name.
+The logical name should be unique across all other connectors, since it is used as a topic name prefix for all Kafka topics that receive records from this connector.
+ +
+[WARNING]
+====
+Changing the value of this property disrupts connector operations.
+All stored offsets are flushed and the connector must re-consume all of the source data.
```

Comment:
This is not quite correct. The connector offsets are stored using the Kafka Connect connector name, i.e. also after updating `database.server.name`, the offsets of that connector would be retrievable. But it wouldn't find its history topic (as the server name is a prefix to that), and all the change event topics would be used with different names too. I'd say something along the lines of

> The value of this property must not be changed. Doing so will result in different topic names being used after a connector restart. [for those connectors where it applies:] Also the connector will not be able to recover its database history topic after a restart.